File: 0485-max-consecutive-ones/0485-max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function (nums) {
    let max = -Infinity, i = 0
    nums.forEach(val => {
        if (val === 1) i++
        else i = 0
        max = Math.max(max, i)
    })
    return max
};

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = Integer.MIN_VALUE, k = 0;
        for (int n : nums)
            if (n == 1)
                k++;
            else {
                max = Math.max(max, k);
                k = 0;
            }
        return Math.max(max, k);
    }
}